### PR TITLE
2022-10-10 weekly report

### PR DESCRIPTION
**Pull request by CohenArthur**

### CODE REVIEW

File: 2022-10-10-report.org [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+| Data Structures 1 - Core          |      100% |      100% | -     | 30th Nov 2020  | 27th Jan 2021   | 29th Jan 2021 |
+| Control Flow 1 - Core             |      100% |      100% | -     | 28th Jan 2021  | 10th Feb 2021   | 26th Feb 2021 |
+| Data Structures 2 - Generics      |      100% |      100% | -     | 11th Feb 2021  | 14th May 2021   | 28th May 2021 |
+| Data Structures 3 - Traits        |      100% |      100% | -     | 20th May 2021  | 17th Sept 2021  | 27th Aug 2021 |
+| Control Flow 2 - Pattern Matching |      100% |      100% | -     | 20th Sept 2021 | 9th Dec 2021    | 29th Nov 2021 |
+| Macros and cfg expansion          |      100% |      100% | -     | 1st Dec 2021   | 31st Mar 2022   | 28th Mar 2022 |
+| Imports and Visibility            |      100% |      100% | -     | 29th Mar 2022  | 13th Jul 2022   | 27th May 2022 |
+| Const Generics                    |       76% |       76% | -     | 30th May 2022  | -               | 17th Oct 2022 |
+| Intrinsics and builtins           |       15% |       15% | -     | 6th Sept 2022  | -               | 14th Nov 2022 |
```
Comment (philberty) [inline 2022-10-10-report.org:77]: Do you want to add in the other milestones we created?

Comment (CohenArthur) [reply 2022-10-10-report.org:76]: Yes, they were still missing some issues up until this morning but that's now done :) I added them in 242f971

**Review by philberty — APPROVED** [review]
LGTM

File: 2022-10-10-report.org [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+- testing: try loop in const function [[https://github.com/rust-gcc/gccrs/pull/1554][PR1554]]
+- Dump item visibility [[https://github.com/rust-gcc/gccrs/pull/1547][PR1547]]
+- Reintroduce targetrustm hooks without the maze of macro definitions in target headers [[https://github.com/rust-gcc/gccrs/pull/1543][PR1543]]
+
+*** Contributors this week
+
+- [[https://github.com/ibuclaw][Iain Buclaw]]
+- [[https://github.com/dafaust][David Faust]]
+- [[https://github.com/abbasfaisal][Faisal Abas]]
+
```
Comment (philberty) [inline 2022-10-10-report.org:36]: I think we missed @dkm lets add him in :D

Comment (dkm) [reply 2022-10-10-report.org:36]: Sure, forgetting such a big contribution would be sad :D Thanks anyway!

**Pull request merged**